### PR TITLE
Customization and some fixes for Blockchain

### PR DESCRIPTION
**Pull request by dolauli**
https://github.com/Azure/azure-powershell/issues/9884
There are still two things need to be done.

1. Implement -Type with enums or switches for the API Test-AzBlockchainLocationNameAvailability, which can not be done is because in the swagger spec, the service team does not define the acceptable values for -Type.

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/BlockChain/custom/New-AzBlockchainTransactionNode.ps1 [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.String]
+    # Gets or sets the transaction node location.
+    ${Location},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.Security.SecureString]
+    # Sets the transaction node dns endpoint basic auth password.
```
Comment (markcowl) [inline src/BlockChain/custom/New-AzBlockchainTransactionNode.ps1]: Should have ValidateNotNullOrEmpty attribute, or the code below should also verify the password is not null or empty

File: src/BlockChain/custom/New-AzBlockchainMember.ps1 [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.String]
+    # The GEO location of the blockchain service.
+    ${Location},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.Security.SecureString]
+    # Sets the basic auth password of the blockchain member.
```
Comment (markcowl) [inline src/BlockChain/custom/New-AzBlockchainMember.ps1]: Should have ValidateNotNullOrEmpty attribute, or the code below should account for case where parameter is bound and null or empty.

File: src/BlockChain/docs/latest-2019-04-30/Get-AzBlockchainConsortium.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+## DESCRIPTION
+Lists the available consortiums for a subscription.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
```
Comment (markcowl) [inline src/BlockChain/docs/latest-2019-04-30/Get-AzBlockchainConsortium.md]: Shoud have examples filled out

File: src/BlockChain/docs/latest-2019-04-30/Get-AzBlockchainMember.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+```
+
+## DESCRIPTION
+Get details about a blockchain member.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
```
Comment (markcowl) [inline src/BlockChain/docs/latest-2019-04-30/Get-AzBlockchainMember.md]: Same comment - applies across the board to all examples - we should have at least one workign example for each cmdlet

File: src/BlockChain/docs/latest-2019-04-30/Update-AzBlockchainTransactionNode.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+Dynamic: False
+```
+
+### -Password
```
Comment (markcowl) [inline src/BlockChain/docs/latest-2019-04-30/Update-AzBlockchainTransactionNode.md]: Don't we need to update this one as wqell to be a secure string?

File: src/BlockChain/docs/latest-2019-04-30/Update-AzBlockchainMember.md [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+Accept pipeline input: False
+Accept wildcard characters: False
+Dynamic: False
+```
+
+### -Password
+Sets the transaction node dns endpoint basic auth password.
+
+```yaml
+Type: System.String
```
Comment (markcowl) [inline src/BlockChain/docs/latest-2019-04-30/Update-AzBlockchainMember.md]: Should be a SecureString

**Comment by dolauli** [comment]
@Erich, If you got a chance, please review it since Mark is on vocation.

**Pull request merged**